--- client/src/pages/Upload.jsx
+++ client/src/pages/Upload.jsx
@@ -25,6 +25,15 @@
     lyricist: '',
     producer: ''
   });
+  const [musicinfo, setMusicinfo] = useState({
+    vocalinstrumental: '',
+    gender: '',
+    speed: '',
+    tags: {
+      genres: [],
+      instruments: []
+    }
+  });

   const [audioUrl, setAudioUrl] = useState('');
   const singleAudioRef = React.useRef(null);
