--- server/models/Track.js
+++ server/models/Track.js
@@ -47,6 +47,15 @@
       lyricist: { type: String, default: '' },
       producer: { type: String, default: '' },
     },
+    musicinfo: {
+      vocalinstrumental: { type: String, default: '' },
+      gender: { type: String, default: '' },
+      speed: { type: String, default: '' },
+      tags: {
+        genres: [{ type: String }],
+        instruments: [{ type: String }],
+      }
+    }
   },
   {
     timestamps: true,
